components.css

```diff
@@ -43,7 +44,7 @@ body{
     display: block;
     height: 100%;
     width: 100%;
-    line-height: 65px;
+    line-height: 50px;
     font-size: 1.2rem;
     color: black;
```

Comment:
use var for color

Reply:
was using this color only once so used it directly